cvat/apps/dataset_manager/project.py

```diff
@@ -139,7 +138,7 @@ def export(self, dst_file: str, exporter: Callable, host: str='', **options):
         )
 
         temp_dir_base = self.db_project.get_tmp_dirname()
```

Comment:
It makes sense to move these temp dirs to /tmp or the new shared cache as well. Just keep in mind that the temp output archive should be on the same disk as the resulting shared cache file, otherwise `os.rename` can't handle it. `os.rename` can be replaced to `shutil.move`, but this function can't overwrite files.